.github/actions/javascript/postTestBuildComment/postTestBuildComment.js

```diff
@@ -0,0 +1,74 @@
+const core = require('@actions/core');
+const {context} = require('@actions/github');
+const GithubUtils = require('../../../libs/GithubUtils');
+
+const PR_NUMBER = core.getInput('PR_NUMBER', {required: true});
+
+const androidSuccess = core.getInput('ANDROID', {required: true}) === 'success';
+const desktopSuccess = core.getInput('DESKTOP', {required: true}) === 'success';
+const iOSSuccess = core.getInput('IOS', {required: true}) === 'success';
+const webSuccess = core.getInput('WEB', {required: true}) === 'success';
+
+const androidLink = androidSuccess
+    ? core.getInput('ANDROID_LINK')
+    : '❌ FAILED ❌';
+const desktopLink = desktopSuccess
+    ? core.getInput('DESKTOP_LINK')
+    : '❌ FAILED ❌';
+const iOSLink = iOSSuccess ? core.getInput('IOS_LINK') : '❌ FAILED ❌';
+const webLink = webSuccess ? core.getInput('WEB_LINK') : '❌ FAILED ❌';
+
+const androidQRCode = androidSuccess
+    ? `![Android](https://api.qrserver.com/v1/create-qr-code/?size=120x120&data=${androidLink})`
+    : "The code can't be generated, because the android build failed";
```

Comment:
(Not a blocker) To make it super clear, I'd have specified "The QR code can't [...]"